securedrop/source_templates/lookup.html

```diff
@@ -94,14 +94,17 @@ <h2 class="headline">{{ gettext('Get Replies') }}</h2>
 
 <hr class="no-line">
 
+{% if new_user %}
 <div class="code-reminder" id="codename-hint">
   <div id="codename-hint-visible">
     <img class="pull-left" src="{{ url_for('static', filename='i/font-awesome/lock-black.png') }}" width="17px" height="20px"> {{ gettext('Remember your codename is:') }}
     <a id="codename-hint-show" class="show pull-right visible-codename" href="#codename-hint-visible">{{ gettext('Show') }}</a>
     <div id="codename-hint-content" class="hidden-codename codename">
-      <p>{{ codename }} <a id="codename-hint-hide" class="pull-right" href="#codename-hint">{{ gettext('Hide') }}</a></p>
+      <a id="codename-hint-hide" class="pull-right" href="#codename-hint">{{ gettext('Hide') }}</a>
+      <p>{{ codename }}</p>
```

Comment:
Note: This change makes it possible to double click the text to select all of it without it selecting the "Hide" button too.